plugins/sigma.plugins.design/sigma.plugins.design.js

```diff
@@ -1001,6 +1001,78 @@
     };
 
     /**
+     * Deletes styles from an element according to the specified options.
+     * If the only the nodeId is specified, this will delete the 'size', 'color', 'type', 'icon' and
+     * 'image'
```

Comment:
"If the `key` parameter is not specified, this will reset all styles on the specified element."

?